src/utlis/utils.js

```diff
@@ -25,3 +25,18 @@ function closeModalOnRemoteClick(evt) {
     closePopup(evt.target.closest(".modal"));
   }
 }
+
+export function toggleSaving(modalId) {
+  const modal = document.querySelector(modalId);
+  const saveButton = modal.querySelector(".form__popup-button");
+
+  if (saveButton.textContent === "Saving...") {
```

Comment:
The function should not know anything about the specific buttons, so it should have the second argument `buttonText`

```js
export function toggleSaving(modalId, loading, buttonText) {
..
if (loading) {
   saveButton.textContent = "Saving...";
} else {
   saveButton.textContent = buttonText;
}
```

or

```js
export function setButtonText(modalId, text) {
  ..
  saveButton.textContent = buttonText;
```